src/ImageSharp/Processing/Processors/Quantization/DefaultPixelSamplingStrategy.cs

```diff
@@ -101,4 +103,55 @@ Buffer2DRegion<TPixel> GetRow(int pos)
             }
         }
     }
+
+    /// <inheritdoc />
+    public IEnumerable<Buffer2DRegion<TPixel>> EnumeratePixelRegions<TPixel>(ImageFrame<TPixel> frame)
```

Comment:
The multi-frame overload has some unit tests, might be worth to test this one too.

Reply:
Will do!